docs/src/main/paradox/http-server.md

```diff
@@ -0,0 +1,135 @@
+HTTP Server logic
+-----------------
+
+The main class, `QuickstartServer`, is runnable because it extends `App`, as shown in the following snippet. 
+This class is intended to "bring it all together", it is the main class that will run the application, as well 
+as the class that should bootstrap all actors and other dependencies (database connections etc). 
+
+@@snip [QuickstartServer.scala]($g8src$/scala/com/lightbend/akka/http/sample/QuickstartServer.scala) { #main-class }
+
+Notice that we've separated out the `UserRoutes` trait, in which we'll put all our actual route definitions.
+This is a good pattern to follow, especially once your application starts to grow and you'll need some form of 
+compartmentalizing them into groups of routes handling specific parts of the exposed API.
+
+
+## Binding endpoints
+
+Each Akka HTTP `Route` contains one or more `akka.http.scaladsl.server.Directives`, such as: `path`, `get`, `post`, `complete`, etc. There is also a [low-level API](http://doc.akka.io/docs/akka-http/current/scala/http/low-level-server-side-api.html) that allows to inspect requests and create responses manually. For the user registry service, the example needs to support the actions listed below. For each, we can identify a path, the HTTP method, and return value:
+
+|Functionality       | HTTP Method | Path       | Returns              |
```

Comment:
Is there an option to update the image file (docs/src/main/paradox/images/hello-akka-http.png) so it reflects the Paths from here ?